examples/authorize/src/main/java/com/dropbox/core/examples/authorize/ScopeAuthorize.java

```diff
@@ -145,8 +145,16 @@ public DbxAuthFinish authorize(DbxAppInfo appInfo) throws IOException {
                 System.exit(1); return null;
             }
 
+            if (!authFinish.getScope().contains("sharing.read")) {
+                System.err.println("You app doesn't have sharing.read scope. Can't finish" +
+                        " " +
+                        "this example.");
+                System.exit(1); return null;
+            }
+
             assert authFinish.getScope().contains("account_info.read");
             assert authFinish.getScope().contains("files.metadata.write");
+            assert authFinish.getScope().contains("sharing.read");
```

Comment:
Is there a sample? Should that be updated? 
Thanks

Reply:
What do you mean by sample?
This **is** the example for testing authorization
If you mean a sample app that this runs against, that's made manually outside of code.

Reply:
Synced offline. confirmed that changing this example here has no effect on the android sample app.